src/Compilers/CSharp/Portable/Symbols/Source/LocalFunctionSymbol.cs

```diff
@@ -493,6 +510,35 @@ public override ImmutableArray<TypeParameterConstraintClause> GetTypeParameterCo
             return _lazyTypeParameterConstraints;
         }
 
+        public override ImmutableArray<CSharpAttributeData> GetAttributes()
+        {
+            var lazyCustomAttributesBag = _lazyCustomAttributesBag;
+            if (lazyCustomAttributesBag != null)
```

Comment:
>!= null [](start = 40, length = 7)

Consider using `== null` and moving `LoadAndValidateAttributes` inside the `if`, so there's only one `return` statement. Same for `GetReturnTypeAttributes`.